sanic/app.py

```diff
@@ -50,13 +50,18 @@ def __init__(self, name=None, router=None, error_handler=None):
         self.debug = None
         self.sock = None
         self.listeners = defaultdict(list)
+        self.is_running = False
 
         # Register alternative method names
         self.go_fast = self.run
 
     @property
     def loop(self):
         """Synonymous with asyncio.get_event_loop()."""
+        if not self.is_running:
```

Comment:
We could but if a user is running an app async chances are they probably already have a global loop that they want to use instead.